nixos/modules/services/misc/jellyfin.nix

```diff
@@ -24,6 +24,38 @@ in
         description = lib.mdDoc "Group under which jellyfin runs.";
       };
 
+      dataDir = mkOption {
+        type = types.path;
+        default = "/var/lib/jellyfin";
+        description = lib.mdDoc ''
+          base data directory, see https://jellyfin.org/docs/general/administration/configuration/#data-directory
+        '';
+      };
+
+      configDir = mkOption {
+        type = types.path;
+        default = "${cfg.dataDir}/config";
```

Comment:
I think we need to define a `defaultText` for this option.

Reply:
amended the commit to address this, thanks